<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="teLog">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>511</width>
      <height>461</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::WidgetWidth</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbLocalIP">
       <property name="text">
        <string>LocalIP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBoxPlus" name="cbbLocalIP">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbLocalPort">
       <property name="text">
        <string>LocalPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLocalPort">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbRemoteIP">
       <property name="text">
        <string>Remote IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRemoteIP"/>
     </item>
     <item>
      <widget class="QLabel" name="lbRemotePort">
       <property name="text">
        <string>Remote Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRemotePort"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnOC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>289</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbModeSelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>289</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="teTXData"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbbTXEndWith"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnTX">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTXDatasOC">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QComboBoxPlus</class>
   <extends>QComboBox</extends>
   <header>utilUiElement.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
